airflow/lineage/__init__.py

```diff
@@ -165,6 +163,10 @@ def wrapper(self, context, *args, **kwargs):
 
         self.outlets.extend(self._outlets)
 
+        # render inlets and outlets
+        self.inlets = [_render_object(i, context)
+                       for i in self.inlets if attr.has(i)]
+
```

Comment:
I think we should use the same convention here for both inlets/outlets rendering - either list comprehension or list(map(filter ? The list comprehension looks quite a bit cleaner IMHO.